packages-ts/gauntlet-terra-contracts/src/commands/contracts/ocr2/inspection/inspect.ts

```diff
@@ -0,0 +1,165 @@
+import { inspection } from '@chainlink/gauntlet-core/dist/utils'
+import { getRDD } from '../../../../lib/rdd'
+import { InspectInstruction, InspectionInput, instructionToInspectCommand } from '../../../abstract/inspectionWrapper'
+import { getOffchainConfigInput, OffchainConfig } from '../setConfig'
+
+type Expected = {
```

Comment:
I'd suggest another name here:

```suggestion
type OnChainData = {
```

Reply:
`Expected` is also the user input, what the `OnChainData` will be checked against.